docs/sources/configuration/_index.md

```diff
@@ -89,6 +89,8 @@ ${VAR:default_value}
 
 Where default_value is the value to use if the environment variable is undefined.
 
+Pass the `-config.expand-env` flag at the command line to enable this way of setting configs.
```

Comment:
Please add an example.

Reply:
@oddlittlebird, You can see the usage example just above this line. It's already there.